src/Elders.Cronus.Persistence.Cassandra/CassandraEventStore.cs

```diff
@@ -128,9 +128,14 @@ public LoadAggregateCommitsResult LoadAggregateCommits(string paginationToken, i
         }
 
         public IEnumerable<AggregateCommit> LoadAggregateCommits(int batchSize)
+        {
+            return LoadAggregateCommitsAsync().Wait().Result;
```

Comment:
@ethno2405 I think that having all items copied into a list inside `Wait` will be slower and memory intensive. May be we could do a test run to compare the original sync version and this one?